timeseries/src/autogluon/timeseries/evaluator.py

```diff
@@ -159,6 +178,12 @@ def _mean_wquantileloss(self, y_true: pd.Series, predictions: TimeSeriesDataFram
             / np.abs(values_true).sum()
         )
 
+    def _wape(self, y_true: pd.Series, predictions: TimeSeriesDataFrame) -> float:
+        y_pred = self._get_median_forecast(predictions)
+        abs_error_sum = abs_error_sum_per_item(y_true=y_true, y_pred=y_pred).sum()
+        abs_target_sum = abs_target_sum_per_item(y_true=y_true).sum()
```

Comment:
Maybe we could just sum the target directly here with `y_true.abs().sum()` without a separate method and a `groupby`?

I think the existing logic where each loss is first computed per item is here in case we at some point let users specify per-item weights (sorry for not documenting the intent in the code 😅). 

But this per-item computation wouldn't be necessary for the absolute target sum because it's just a normalizing constant, so we can just compute the sum directly.

Reply:
Agreed and updated.